app-datepicker.html

```diff
@@ -1077,7 +1077,7 @@
           if (_maxDateObj) {
             _isMoreThanMaxDate = _currentDate.getTime() > _maxDateObj.getTime();
           }
-          _isDisableDates = this.disableDates.some(function(date) {
+          _isDisableDates = !this.disableDates || this.disableDates.some(function(date) {
```

Comment:
Whoops, I reversed the logic I intended.
Although that line should actually be.
`this.disableDates && this.disableDates.length && this.disableDates.some ...`